### DISCUSSION TITLE
Como conseguir insignias

### DISCUSSION THREAD

**Discussion by DonPachos** [community/community#132932] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Como conseguir insignias de la comunidad para mi perfil?

**Comment by SamuelOUS** [comment, marked as answer]
este link te puede ayudar :)
https://github.com/Thinkright20/Profile-Badges

**Reply by DonPachos** [reply]
Gracias!

**Comment by ebndev** [comment]
Hi @DonPachos, thanks for participating!

Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting to ensure discussions are adhering to our [Code of Conduct](https://github.com/community/community/blob/920a928ffe3ae4863b64a4ea26917046e11f5c1a/CODE_OF_CONDUCT.md). 

We’ll be locking this and other related posts for now, but please feel encouraged to create a new discussion in English if you have questions in the future!
